Configuration/HLT/python/autoCondHLT.py

```diff
@@ -1,5 +1,7 @@
 # https://cms-conddb.cern.ch/browser/#search
 
+snapshotTime = "9999-12-31 23:59:59.000"
```

Comment:
summary of the discussion this morning with PPD and AlCa: can you change this from a "future" timestamp to the present ?
Something like
```
snapshotTime = "2015-10-30 00:00:00.000"
```
?
This will restore the reproducibility of the tests.

Of course, this should be updated for new customisations.